app/domain/breadcrumb_trail.rb

```diff
@@ -1,29 +1,38 @@
 class BreadcrumbTrail
-  def self.build(item, category_tree)
-    categories = case item
-      when Core::Category
-        if item.parent_id.present?
-          RootToNodePath.build(item, category_tree)[0...-1]
-        else
-          [HomeCategory.new]
-        end
-      when ToolCategory
-        RootToNodePath.build(item, category_tree)
-      when Core::StaticPage
+  def self.build(item, category_tree = nil)
+    case item
+    when Mas::Cms::Category
+      # recurse and find categories instead of passing in category_tree?
+      if item.parent_id.present?
```

Comment:
could we move this if block into its own method? It will keep the case statement a little cleaner, wdyt?